tests/rules/test_docker_login.py

```diff
@@ -1,5 +1,6 @@
 from thefuck.rules.docker_login import match, get_new_command
 from thefuck.types import Command
+from thefuck.shells import shell
```

Comment:
No need to use `and_` in tests, as the [shell is set to `Generic`](https://github.com/nvbn/thefuck/blob/7f97818663de9351ac7e2c6314ed94184811d62a/tests/conftest.py#L7).

```suggestion
```